packages/web-components/src/accordion-item/accordion-item.ts

```diff
@@ -0,0 +1,40 @@
+import { attr } from '@microsoft/fast-element';
+import { FASTAccordionItem } from '@microsoft/fast-foundation';
+import { AccordionItemExpandIconPosition, AccordionItemSize } from './accordion-item.options.js';
+
+/**
+ * @internal
+ */
+export class AccordionItem extends FASTAccordionItem {
+  /**
+   * Defines accordion header size.
+   *
+   * @public
+   * @default 'medium'
+   * @remarks
+   * HTML Attribute: appearance
+   */
+  @attr({ attribute: 'size' })
```

Comment:
```suggestion
  @attr
```

No need to set attribute if the name is the same.